lib/Doctrine/DBAL/Schema/SchemaDiff.php

```diff
@@ -165,6 +164,6 @@ protected function _toSql(AbstractPlatform $platform, $saveMode = false)
             $sql = array_merge($sql, $platform->getAlterTableSQL($tableDiff));
         }
 
-        return $sql;
+        return array_values($sql);
```

Comment:
Is it still needed? I'd rather mark the required `AbstractPlatform` methods as returning `array<int, string>` (even though that's a different namespace) than call `array_values()` at runtime. I want to avoid the introduction of unnecessary `array_values()` because later, after all types are settled, it will be hard to figure out which of them are no longer needed.

Reply:
Should be able to kill it. I'll make another commit.

Reply:
Fixed with https://github.com/doctrine/dbal/pull/3565/commits/d382e342d37227b734b0b18183a3d3fcd8923ebc